android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -310,3 +346,5 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
         private val kDefaultObjectPosition = Float3(0.0f, 0.0f, -4.0f)
     }
 }
+
+typealias ResourceTask = AsyncTask<Unit, Unit, Map<String, Buffer>>
```

Comment:
`AsyncTask` is deprecated and shouldn't be used anymore. You should look into coroutines instead since this is a Kotlin library.

Reply:
Done, added an impl dep on kotlinx-coroutines too.